x-pack/platform/plugins/shared/alerting/server/alerts_service/lib/create_concrete_write_index.ts

```diff
@@ -96,6 +104,35 @@ const updateUnderlyingMapping = async ({
 
     return;
   } catch (err) {
+    if (attempt <= MAX_FIELDS_LIMIT_INCREASE_ATTEMPTS) {
```

Comment:
Seems like it might be nice to have a debug log here, indicating what's going on, attempt #, and error.

Reply:
Added a debug log